### PR TITLE
MSA updates

### PR DESCRIPTION
**Pull request by JTPetter**
- Updated gauge r&R non. rep. help file.
- Gauge r&R linearity change default value of process variation to 1.
- Fix capitalization, aligned naming, and changed option names as suggested by Rene.
- Type1 Study: Changing percent of tolerance for Cg now affects boundaries displayed in plot.
- Type 1 Study: Added tolerance bounds to table.
- Type 1 Study: Added mean and three sigma limits to run chart.
- Type 1 Study: Added option to select left- or right-open bins and centered bins on integers.
- Gauge r&R Crossed: Changed order of variance components to display operator x part before part-to-part.
- Gauge r&R Crossed: Fixed bug where variance component table would display empty cells when historical std. dev. was used.
- Gauge r&R Nested: Implemented traffic light chart for regular output and report.
- Gauge r&R Crossed & Nested: Fixed crash of range chart and average chart per operator if only on measurement per part was supplied.
- Gauge Linearity: Changed order of Std. Error and t-statistic in regression table.
- Gauge Linearity: Added regression equation to output and made percent process variation graph in JASP style.
- Gauge r&R crossed and nested: Added gauge evaluation tables and number of distinct categories to report
- Gauge r&R crossed: Made more robust towards input that has no repeated measurements.
- Gauge r&R nested: Fixed calculation to always recognize nested structure.
- All MSA: Added error messages for missing values that crash analyses.
- Attribute Agreement: Fix crash when no standards were provided.
- Added verified unit tests.

The large number of changed lines is related to the updated unit tests (many new plot files).

### CODE REVIEW

File: R/msaGaugeLinearity.R [hunk 1]
```diff
@@ -151,6 +154,7 @@ msaGaugeLinearity <- function(jaspResults, dataset, options, ...) {
     linearity <- abs(coefficientSlope) * options[["manualProcessVariationValue"]]
     percentLin <- (linearity / options[["manualProcessVariationValue"]]) * 100
     plusOrMin <- if (coefficientSlope > 0) "+" else "-"
+    regressionEquation <- gettextf("Bias = %.2f %s %.2f * Reference value", coefficientConstant, plusOrMin, abs(coefficientSlope))
```
Comment (vandenman) [inline R/msaGaugeLinearity.R:157]:
```suggestion
    regressionEquation <- gettextf("Bias = %1$.2f %2$s %3$.2f * Reference value", coefficientConstant, plusOrMin, abs(coefficientSlope))
```

File: inst/qml/msaGaugeRR.qml [hunk 2]
```diff
@@ -129,7 +129,7 @@ Form
 
 	Group
 	{
-		title: 									qsTr("ANOVA Method Options")
+		title: 									qsTr("Analysis ptions")
```
Comment (vandenman) [inline inst/qml/msaGaugeRR.qml:132]:
```suggestion
		title: 									qsTr("Analysis options")
```

File: inst/qml/msaGaugeRRnonrep.qml [hunk 3]
```diff
@@ -105,127 +105,128 @@ Form
...
 
 			DoubleField
 			{
-				name:							"historicalSdValue"
-				label:							qsTr("Std. Deviation value")
-				defaultValue:					3
-				enabled:						variationReference.currentValue == "historicalSd"
+				name: 						"toleranceValue"
+				defaultValue:				1
+				min:						0.000000001
```
Comment (vandenman) [inline inst/qml/msaGaugeRRnonrep.qml:142]: why not
```suggestion
				min:						0
				inclusive:					JASP.MaxOnly
```
?

See https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-guide.md#doublefield for the inclusive option and its possible values.

**Review by vandenman — CHANGES_REQUESTED** [review]
some minor comments below. Also, it's easier to review multiple smaller PRs than one big one!

File: inst/qml/msaGaugeRRnonrep.qml (hunk 3 above)
Comment (JTPetter) [reply inst/qml/msaGaugeRRnonrep.qml:142]: Yes, thanks!

**Comment by JTPetter** [comment]
Thanks - will try to make smaller PRs whenever possible.

**Comment by JTPetter** [comment]
Added fix for https://github.com/jasp-stats/jasp-issues/issues/2867

**Review by vandenman — APPROVED** [review]

**Pull request merged**